Kitodo/src/main/resources/hibernate.cfg.xml

```diff
@@ -52,7 +52,7 @@
         <property name="hibernate.cache.region.factory_class">org.hibernate.cache.ehcache.EhCacheRegionFactory</property>
 
         <!-- Enable Hibernate's automatic session context management -->
-        <property name="current_session_context_class">managed</property>
+        <property name="current_session_context_class">thread</property>
```

Comment:
What is the reason to switch from `managed` to `thread`? What kind of issue did you fix with this change?

Reply:
thread: current sessions are tracked by thread of execution.

managed: current sessions are tracked by thread of execution. However, you are responsible to bind and unbind a Session instance with static methods on this class: it does not open, flush, or close a Session.

I suspect that thread is more forgiving than managed...

Reply:
Thank you for the explanation.